src/crypto/crypto_sig.cc

```diff
@@ -753,6 +758,7 @@ Maybe<bool> SignTraits::AdditionalConfig(
     ManagedEVPPKey m_pkey = params->key->GetAsymmetricKey();
     Mutex::ScopedLock lock(*m_pkey.mutex());
     if (EVP_PKEY_id(m_pkey.get()) == EVP_PKEY_EC) {
+      Mutex::ScopedUnlock unlock(lock);
```

Comment:
> Is this because something in `ConvertFromWebCryptoSignature()` locks the mutex again?

Yeah, currently `ConvertFromWebCryptoSignature` will call `GroupOrderSize` which will also acquire the lock. But looking at this again (the unlocking was a change made yesterday) it should be possible to not acquire the lock in `GroupOrderSize` and remove the unlocking from the callers. I'll revisit this today.

And thanks for reviewing despite the size of this PR.

Reply:
I've removed the unlock calls now, would you be able to take a look and see what you think?